plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/page/component/TextAndButtonComponent.java

```diff
@@ -54,6 +53,11 @@ public void setText(String value) {
     text.setText(value);
   }
 
+  public boolean isEnabled() {
```

Comment:
This is used only in tests. Probably don't have to be `public` but `@VisibleForTesting`.